src/Place/Commands/MarkAsDuplicate.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace CultuurNet\UDB3\Place\Commands;
+
+use CultuurNet\UDB3\Offer\Commands\AuthorizableCommandInterface;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+
+final class MarkAsDuplicate implements AuthorizableCommandInterface
```

Comment:
I'm not sure if we should implement `AuthorizableCommandInterface` already if we don't know the right permission yet and the command will only be available from the CLI at first? 🤔 
But if you feel like this is the safest way to prevent it from accidentally being dispatched from an HTTP controller or other place I'm fine with keeping it.

Reply:
It will work either way and it doesn't stop us from dispatching in through CLI so I thought it was a quick win to already implement it. Changing the permission later is always an option.